web-console/1-homepage/project-panel.md

```diff
@@ -1,63 +1,98 @@
+- On day 1, the projects panel includes the following sections in this order:
+  1. **Create Project Button** (optional): The create project button is displayed by default, but admin users may choose to hide this button if desired.
+  2. **Guided Tour:** Action button that launches a quick tutorial to identify key sections of the interface.
+    * Details TBD
+  3. **Resources:** Additional links that may help user learn more about the product will be listed here.
+    * Details TBD
+  4. **Custom Text/Links** (optional): This section is optional and configurable by the admin user. It allows admin users to provide additional content for their day-to-day users.
 
-# Template
+- The "Getting Started" section will no longer appear once the user has at least one project created.
+- The project panel will then display the Day 2 Experience detailed below.
 
-- This is a description of the section and where it is located/how to get to it
-- This is a description of what this section is for
+### Implementation Details
+- The **Custom Text/Links** are configurable by an admin user with full DOM access to this section only.
+- The **Guided Tour** will use the [Hopscotch framework](http://linkedin.github.io/hopscotch/)
 
+### Customer Feedback
+- TBD
 
-## Use Case 1 (e.g. Browse for service)
 
-![template](img/template-01.png)
-- description of what is happening in this wireframe
+## Day 2 Experience
 
-#### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
+### Create Project
+![template](img/OpenShiftMall_Day1-ProjectsBar-2B.png)
 
-![template](img/template-02.png)
-- description of what is happening in this wireframe
+- The create project button will open an overlay panel with a three-field form inside. See implementation details for more info.
+- Once the user has at least one project created, the "My Projects" section will appear at the top of the panel. The Create Project button (if being used) will move down under "My Projects".
+- The Getting Started section should move below the first project until resources are added to the project.
 
-#### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
+![template](img/OpenShiftMall_Day1-ProjectsBar-3B.png)
 
-#### Customer Feedback (if applicable)
-- Customer Feedback Received
-  - Overview of customer feedback received
-- Necessary Customer Feedback
-  - Questions to follow up with customers
+- Once created, the project will be listed as a card at the top of the projects list, in an "empty state."
+- The empty state is defined as a new project which does not have any resources and only has a name, date, and description.
+- **Kebab Actions:** There are three actions under the kebab menu for projects: edit details, edit membership, and delete. These actions should function as they currently behave today. The delete action opens a confirmation modal and the other two navigate to a new page.
 
+### Single Project
```

Comment:
@beanh66 Can you take a look at how projects are currently shown in OpenShift, I assume we should be using the same method of displaying timestamp ... they are doing something like "x days ago" or actual date